test/distrib/python/test_packages.sh

```diff
@@ -28,10 +28,12 @@ else
   echo "Testing Python source distribution"
   ARCHIVES=("$EXTERNAL_GIT_ROOT"/input_artifacts/grpcio-[0-9]*.tar.gz)
   TOOLS_ARCHIVES=("$EXTERNAL_GIT_ROOT"/input_artifacts/grpcio-tools-[0-9]*.tar.gz)
-  HEALTH_ARCHIVES=("$EXTERNAL_GIT_ROOT"/input_artifacts/grpcio-health-checking-[0-9]*.tar.gz)
-  REFLECTION_ARCHIVES=("$EXTERNAL_GIT_ROOT"/input_artifacts/grpcio-reflection-[0-9]*.tar.gz)
 fi
 
+HEALTH_ARCHIVES=("$EXTERNAL_GIT_ROOT"/input_artifacts/grpcio-health-checking-[0-9]*.tar.gz)
```

Comment:
Why are we moving this out of the ```"binary"``` ```"source"``` checks?

Reply:
I think it's better to do it on both binary/source. Those are source-only packages so it wouldn't matter for them anyway, but our users might install them alongside the binary `grpcio` vs source `grpcio` and that might be a slightly different environment, (and the test is cheap), so I'd rather run it on every case.